packages/govern-server/src/core/data/queries/fragments.ts

```diff
@@ -0,0 +1,133 @@
+import gql from 'graphql-tag'
```

Comment:
Not sure if this dependency is required cause ``apollo-server`` does expose this dependency as well.. [see](https://github.com/aragon/govern/blob/master/packages/govern-server/src/api-graphql/index.ts#L1)..

Reply:
I think we should keep everything in `core/` separate from `apollo-server`, as this is the part that is going to move in a separate package and run in the browser eventually.

Reply:
I think then should we probably move the core out in a separate package if I got you right?

btw.: I was only seeing here the duplicated dependency ``graphql-tag``. 🤷‍♂️

Reply:
> I think then should we probably move the core out in a separate package if I got you right?

That’s exactly the idea: starting with `core/` in this package so we don’t spend any time on this separation at first, then move it to its own package to separate the server app from the core library.

Reply:
As discussed on the call: I agree to move the core out of the server package as well because of the fully decentralized version of govern in the future. Means we can keep this dependency for now.